### DISCUSSION TITLE
Make text copyable

### DISCUSSION THREAD

**Discussion by Tealk** [thunder-app/thunder#161] [Ideas]
Is there a reason why texts cannot be copied in the app?

**Comment by hjiangsu** [comment]
This is generally just something to do with how Flutter handles copying and pasting text

You need to specifically let Flutter know which texts are copyable and then it'll generally allow you to do that (maybe with some exceptions)

If you have any specific parts of the app that you want to be able to copy text from, let me know and I'll see if its possible!

**Reply by Tealk** [reply]
Mainly posts and comments

**Reply by hjiangsu** [reply]
That should be possible! Could you create a new issue for this? Thanks!

**Comment by Tealk** [comment]
Create issue #162
